openmdao/docs/usr-guide/tutorials/recording.rst

```diff
@@ -352,7 +325,7 @@ etc. To access the data from our run, we can use the following code:
     top.setup()
     top.run()
 
-    top.driver.recorders[0].close()
+    top.cleanup()
 
 .. testoutput:: reading
    :hide:
```

Comment:
I think we should recommend that they always do it.  It certainly doesn't hurt anything.